tests/test_2022_11_15.py

```diff
@@ -0,0 +1,143 @@
+"""
+Tests for voting 15/11/2022.
+"""
+from scripts.vote_2022_11_15 import start_vote
+from utils.test.tx_tracing_helpers import *
+from utils.test.event_validators.payout import (
+    Payout,
+    validate_token_payout_event,
+)
+from utils.finance import ZERO_ADDRESS
+from brownie.network.transaction import TransactionReceipt
+from brownie import interface
+
+
+dao_agent_address = "0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c"
+lido_dao_token = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32"
+weth_token = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+dai_token_address = "0x6B175474E89094C44Da98b954EedeAC495271d0F"
```

Comment:
I think these addresses are present in `utils/mainnet_config.py`. I'd suggest using that file as the single source of truth.

Reply:
I would rather hardcoded them in tests indeed (because config can be changed remain unnoticed)

Reply:
Okay, yeah, I see where you are coming from :) But if we don't trust the global config file, that kinda defeats the whole purpose of having one.

Reply:
The global config is a good thing when you write scripts and utils. Though, everything should have a fuse (especially, if we are transferring funds). That's why I propose to have the hardcoded test constants (i.e., break the circuit if something is wrong).